### DISCUSSION TITLE
Concentrado de links

### DISCUSSION THREAD

**Discussion by CarloAyala** [Launch-X-Latam/MisionFrontEnd#35] [Show and tell]
Hola compañeros, les comparto un concentrado de links que nos pueden ayudar a tener la información mas accesible.

<a href="https://web.microsoftstream.com/channel/893be8b9-bab6-4ea6-9ab0-2d5874f9ab1f">
      <img src="https://img.shields.io/badge/Teams%20-Sesiones%20Grabadas%E2%86%92-gray.svg?colorA=655BE1&colorB=4F44D6&style=for-the-badge"/>
</a>
<br><br>
<a href="https://github.com/Launch-X-Latam/MisionFrontEnd/tree/main/01%20-%20INTRO">
      <img src="https://img.shields.io/badge/GitHub-Material%20Semana%201%E2%86%92-gray.svg?colorA=606060&colorB=C8C8C8&style=for-the-badge"/>
</a>
<br>
<a href="https://github.com/Launch-X-Latam/MisionFrontEnd/tree/main/02%20-%20HTML">
      <img src="https://img.shields.io/badge/GitHub-Material%20Semana%202%E2%86%92-gray.svg?colorA=606060&colorB=C8C8C8&style=for-the-badge"/>
</a>
<br>
<a href="https://github.com/Launch-X-Latam/MisionFrontEnd/tree/main/03%20-%20CSS">
      <img src="https://img.shields.io/badge/GitHub-Material%20Semana%203%E2%86%92-gray.svg?colorA=606060&colorB=C8C8C8&style=for-the-badge"/>
</a>
<br>
<a href="https://github.com/Launch-X-Latam/MisionFrontEnd/tree/main/04%20-%20JS">
      <img src="https://img.shields.io/badge/GitHub-Material%20Semana%204%E2%86%92-gray.svg?colorA=606060&colorB=C8C8C8&style=for-the-badge"/>
</a>
<br>
<a href="https://github.com/Launch-X-Latam/MisionFrontEnd/tree/main/05%20-%20VUEJS">
      <img src="https://img.shields.io/badge/GitHub-Material%20Semana%205%E2%86%92-gray.svg?colorA=606060&colorB=C8C8C8&style=for-the-badge"/>
</a>
<br>
<a href="https://github.com/Launch-X-Latam/MisionFrontEnd/tree/main/06%20-%20VUEX">
      <img src="https://img.shields.io/badge/GitHub-Material%20Semana%206%E2%86%92-gray.svg?colorA=606060&colorB=C8C8C8&style=for-the-badge"/>
</a>
<br><br>
<a href="https://learn.microsoft.com/en-gb/training/challenges?id=a48392e5-c90b-416f-b76a-4b92cf84e9a5&WT.mc_id=cloudskillschallenge_a48392e5-c90b-416f-b76a-4b92cf84e9a5&source=learn">
      <img src="https://img.shields.io/badge/Microsoft-Cloud%20Skills%20Challenge%E2%86%92-gray.svg?colorA=F74E1E&colorB=F67753&style=for-the-badge"/>
</a>
<br>
<a href="https://info.microsoft.com/LA-DevEngage-WBNR-FY22-02Feb-15-Launch-X-SRDEM100737_LP02-On-Demand-Registration---Form-in-Body.html">
      <img src="https://img.shields.io/badge/Microsoft-Formulario%20de%20acceso%20al%20curso%E2%86%92-gray.svg?colorA=F74E1E&colorB=F67753&style=for-the-badge"/>
</a>

**Comment by JosafatJimenezB** [comment]
Donde generaste los batch?

**Reply by CarloAyala** [reply]
Cuál batch?
 ![tenor.gif](https://user-images.githubusercontent.com/114269986/192654978-b3f274cc-afe5-491c-938c-7d3b5fb03da7.gif)

**Reply by JosafatJimenezB** [reply]
para poner los links asi, se ven bien cool!

**Reply by CarloAyala** [reply]
a mira esta bien sencillo, solo agregue un hipervínculo con una imagen, igual que en HTML, así:

`<a href="https://web.microsoftstream.com/channel/893be8b9-bab6-4ea6-9ab0-2d5874f9ab1f">
      <img src="https://img.shields.io/badge/Teams%20-Sesiones%20Grabadas%E2%86%92-gray.svg?colorA=655BE1&colorB=4F44D6&style=for-the-badge"/>
</a>`

Y esas "imágenes" las puedes generar aquí [Shields](https://shields.io/), hay diferentes estilos y puedes cambiarles el texto y los colores.
Imágenes entre comillas porque creo que se llaman **Badges**

**Reply by JosafatJimenezB** [reply]
andale a esas me referia, gracias por la info

**Comment by D3r3-k** [comment]
Para que sirve el cloud skills challenge?

**Reply by CarloAyala** [reply]
Hola compañero, el challenge es para certificarte, al final del curso nos van a dar un reconocimiento pero opcionalmente puedes realizar el challenge del link para que puedas certificar, es opcional pero tiene más peso que el reconocimiento.

**Reply by JosafatJimenezB** [reply]
pasen el link compañeros explorers

**Reply by AlanGRuiz** [reply]
Sí es para certificarte pero en Azure AZ-900, por lo que entiendo es un curso especializado en esa certificación.

**Reply by JosafatJimenezB** [reply]
el link esta en teams?

**Reply by nicolaaasss** [reply]
para hacerlo es necesario el correo de innovacion.mx?

**Reply by CarloAyala** [reply]
@JosafatJimenezB es el que dice Cloud Skill Challenge de color naranja aquí arribita.

**Reply by CarloAyala** [reply]
@nicolaaasss no estoy muy seguro si se pueda pero a mi me dejo con mi correo personal.

**Reply by JosafatJimenezB** [reply]
gracias @CarloAyala

**Comment by Ignacioskm** [comment]
Muchas gracias compañero!

**Comment by Keystle** [comment]
Muchas gracias por la data

**Comment by LaMazorca** [comment]
Gracias amigo

**Comment by 0523Alfonso** [comment]
Muchas gracias

**Comment by ght021** [comment]
Gracias!👍
